internal/service/plan.go

```diff
@@ -650,84 +635,44 @@ func (s *planService) DeletePlan(ctx context.Context, id string) error {
 	return nil
 }
 
-func (s *planService) SyncPlanPrices(ctx context.Context, id string) (*SyncPlanPricesResponse, error) {
+func (s *planService) SyncPlanPrices(ctx context.Context, id string) (*dto.SyncPlanPricesResponse, error) {
```

Comment:
Need very very strong set of test cases specifically for the part where the whole logic of create/update/terminate is listed